deps.sh

```diff
@@ -27,26 +27,20 @@ get_deps() {
     done
 }
 
-if [[ -z "${DEP_REPO:-}" ]]; then
-    DEP_REPO="github.com/scootdev/scoot"
-    scootdev="${GOPATH}/src/$(dirname ${DEP_REPO})"
-    mkdir -p "${scootdev}" && cd "${scootdev}"
-    git clone "https://${DEP_REPO}"
-else
-    depdir="${GOPATH}/src/$(dirname ${DEP_REPO})"
-    mkdir -p "${depdir}" && cd "${depdir}"
-    cp -r "${GOPATH_ORIG}/src/${DEP_REPO}" ./
-fi
+depRepo="github.com/twitter/scoot"
+depDir="${GOPATH}/src/$(dirname ${depRepo})"
+mkdir -p "${depDir}" && cd "${depDir}"
+cp -R "${SCOOT_ORIG}" "./$(basename ${depRepo})"
 
 echo "Darwin Deps."
 export GOOS=darwin GOARCH=amd64
-get_deps "${DEP_REPO}"
+get_deps "${depRepo}"
 
 echo "Windows Deps."
 export GOOS=windows GOARCH=amd64
-get_deps "${DEP_REPO}"
+get_deps "${depRepo}"
 
-HANDLED=$(find ${GOPATH} -name .git | sort | uniq | sed -E "s,${GOPATH}/src/|/\.git,,g" | grep -v scootdev)
+HANDLED=$(find ${GOPATH} -name .git | sort | uniq | sed -E "s,${GOPATH}/src/|/\.git,,g" | grep -v twitter)
```

Comment:
I don't really  understand this script, but... is this change supposed to be in this repo (scootdev/scoot) or in the new twitter/scoot repo?

Reply:
Nope, I made this in the twitter repo and then thought better of it and moved it here.